_components/footers.md

```diff
@@ -6,16 +6,244 @@ title: Footers
+            <li><a href="javascript:void(0)">Secondary link</a></li>
+          </ul>
+        </nav>
+
+        <div class="usa-sign_up-block width-one-third">
+          <h3 class="usa-sign_up-header">Sign up</h3>
+
+          <label class="" for="email" id="">Your email address</label>
+          <input id="email" name="email" type="input-type-email">
+
+          <button type="submit">Sign up</button>
+        </div>
+      </div>
+    </div>
+
+    <div class="usa-footer-secondary-section usa-footer-big-secondary-section">
+      <div class="grid-box">
+        <div class="usa-footer-logo width-one-half">
+          <img class="usa-footer-logo-img" src="{{ site.baseurl }}/assets/img/logo-img.png" alt="Logo image">
+          <h3 class="usa-footer-logo-heading">Name of Agency</h3>
+        </div>
+
+        <div class="usa-footer-contact-links width-one-half">
+          <div class="usa-social-links">
+            <a href="javascript:void(0)">
+              <svg width="26" height="39" role="img" aria-label="Facebook">
+                <title>Facebook</title>
+                <image xlink:href="{{ site.baseurl }}/assets/img/social-icons/svg/facebook25.svg" src="{{ site.baseurl }}/assets/img/social-icons/png/facebook25.png" width="26" height="39" />
+              </svg>
+            </a>
+            <a href="javascript:void(0)">            
+              <svg width="26" height="39" role="img" aria-label="Twitter">
+                <title>Twitter</title>
+                <image xlink:href="{{ site.baseurl }}/assets/img/social-icons/svg/twitter16.svg" src="{{ site.baseurl }}/assets/img/social-icons/png/twitter16.png" width="26" height="39" />
+              </svg>
+            </a>
+            <a href="javascript:void(0)">
+              <svg width="26" height="39" role="img" aria-label="YouTube">
+                <title>YouTube</title>
+                <image xlink:href="{{ site.baseurl }}/assets/img/social-icons/svg/youtube15.svg" src="{{ site.baseurl }}/assets/img/social-icons/png/youtube15.png" width="26" height="39" />
+              </svg>
+            </a>
+            <a href="javascript:void(0)">
+              <svg width="26" height="39" role="img" aria-label="RSS">
+                <title>RSS</title>
+                <image xlink:href="{{ site.baseurl }}/assets/img/social-icons/svg/rss25.svg" src="{{ site.baseurl }}/assets/img/social-icons/png/rss25.png" width="26" height="39" />
+              </svg>
+            </a>
+          </div>
+          
+          <h3 class="usa-footer-contact-heading">Agency Contact Center</h3>
+          <p class="usa-footer-phone-number">(800) CALL-GOVT</p>
+          <a href="mailto:javascript:void(0)">info@agency.gov</a>
+        </div>
+      </div>
+    </div>
+  </footer>
+
+  <h3>Footer Medium</h3>
+
+  <footer class="usa-footer usa-footer-medium usa-sans" role="contentinfo">
+    <div class="grid-box usa-footer-return-to-top">
+      <a href="#usa-top-of-page">Return to top</a>
```

Comment:
@maya I remember you had the concern that if a user didn't add this `usa-top-of-page` id to their body, this wouldn't work. Maybe we should just make this `href="@"`?